app/models/security_policy.rb

```diff
@@ -13,7 +13,7 @@ class SecurityPolicy < ApplicationRecord
   has_many :security_policy_rules, :foreign_key => :security_policy_id, :dependent => :destroy
   alias rules security_policy_rules
```

Comment:
Not for this PR (since for backport I'd like to limit the scope of the PR), but is this alias then no longer needed?

Reply:
There are ~6k `\brules\b` in our ruby files.
I didn't know where to even start determining if `rules` was used.